comments.go

```diff
@@ -47,6 +47,32 @@ func (c *CommentsController) Show(ctx *app.ShowCommentsContext) error {
 	})
 }
 
+// Update does PATCH comment
+func (c *CommentsController) Update(ctx *app.UpdateCommentsContext) error {
+	id, err := uuid.FromString(ctx.ID)
+	if err != nil {
+		jerrors, _ := jsonapi.ErrorToJSONAPIErrors(goa.ErrUnauthorized(err.Error()))
+		return ctx.BadRequest(jerrors)
+	}
+
+	return application.Transactional(c.db, func(appl application.Application) error {
+		cm, err := appl.Comments().Load(ctx.Context, id)
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, err)
+		}
+		cm.Body = *ctx.Payload.Data.Attributes.Body
```

Comment:
Hi @baijum ,
You probably would have figured this out already, I was looking at the code and here's how you could get the token ---> user.

1. Get token string from request header , would be in the form "bearer XXXXXXX"
2. Call login.ReadTokenManagerFromContext(..) to get `tokenManager`
3. Use the `tokenManager` object to call `Extract(string)` to get the `account.Identity` object.